src/shared/specifier.c

```diff
@@ -95,10 +95,9 @@ int specifier_printf(const char *text, const Specifier table[], void *userdata,
 
                                         ret = n;
                                         t = n + j + k;
-                                } else {
-                                        *(t++) = '%';
-                                        *(t++) = *f;
-                                }
+                                } else
+                                        /* Oops, an unknown specifier. */
+                                        return -EINVAL;
```

Comment:
urks. that's likely going to break some user code... I figure this as least needs some NEWS coverage...